torch/_logging/_registrations.py

```diff
@@ -45,6 +45,11 @@
     "aot_joint_graph",
     "Print FX joint graph from AOTAutograd, prior to partitioning. Useful for debugging partitioning",
 )
+register_artifact(
+    "compiled_autograd",
+    "Prints various logs in compiled_autograd, including but not limited to the graphs. Useful for debugging compiled_autograd.",
```

Comment:
Nit: as you picked this as an artifact, it would be more symmetric with the other existing logs to name it `compiled_autograd_graph` and have it only print the graph.

Reply:
Good nit. I considered it. I'm gonna use it for more than that, and I always want the graph.